### PR TITLE
feat(maps): add map of Moldova

### PR DESCRIPTION
**Pull request by mihai-ro**

### CODE REVIEW

**Comment by mihai-ro** [comment]
@VictorCazanave , please review this at your earliest convenience. As soon as this is merged, I will put up two other PRs, if that is fine with you as well :)

**Comment by VictorCazanave** [comment]
Thanks a lot for your contribution @mihai-ro! 😃

I'll try to review it this week.

**Comment by wdarins** [comment]
> Thanks a lot for your contribution @mihai-ro! 😃
> 
> I'll try to review it this week.

any news on this?

**Comment by mihai-ro** [comment]
@VictorCazanave , any news on your side about this PR?

File: packages/moldova/README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+- Rîşcani
+- Şoldăneşti
+- Sîngerei
+- Unitatea Teritorială din Stînga Nistrului
+- Soroca
+- Străşeni
+- Ştefan Vodă
+- Taraclia
+- Teleneşti
+- Ungheni
```
Comment (VictorCazanave) [inline packages/moldova/README.md:43]: Is it an up-to-date map?
According to [wikipedia](https://en.wikipedia.org/wiki/Administrative_divisions_of_Moldova) these are actually districts, municipalities and autonomous territories, but it doesn't contain [Transnistria](https://en.wikipedia.org/wiki/Transnistria_autonomous_territorial_unit), and [Tighina](https://en.wikipedia.org/wiki/Tighina_County_(Moldova)) is an old county.

File: packages/moldova/README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+`yarn add @svg-maps/moldova`
+
+## Credits
+
+This map is based on the work of [MapSVG](https://mapsvg.com).
+
+The original map is available [here](https://mapsvg.com/maps/moldova) under the [Creative Commons Attribution 4.0 International](https://creativecommons.org/licenses/by/4.0/) license.
+
+### Changes
```
Comment (VictorCazanave) [inline packages/moldova/README.md:61]: Could you list the changes made on the original map please?

**Review by VictorCazanave — COMMENTED** [review]
Thank you for you patience!
Just 1 question and 1 small fix, then I'll glad to merge it.

File: packages/moldova/README.md (hunk 2 above)
Comment (mihai-ro) [reply packages/moldova/README.md:61]: @VictorCazanave please review

File: packages/moldova/README.md (hunk 1 above)
Comment (mihai-ro) [reply packages/moldova/README.md:43]: @VictorCazanave  I used the original svg maps; you are right about Tighina - this is the old name of the county, so I replaced it with the current name - Bender.
Transnistria is already listed on the map, except officially (in the docs) the Moldovan state calls it ”Unitatea Teritorială din Stânga Nistrului”. It is more of a political statement, as Transnistria is not recognised as a state; similarly to ”UTA Găgăuză” I extended the label and simply added ”(Transnistria)” in brackets. 
Please review

**Comment by VictorCazanave** [comment]
Thanks again!

@all-contributors please add @mihai-ro for code

**Pull request merged**